packages/web-components/src/rating-display/rating-display.ts

```diff
@@ -0,0 +1,122 @@
+import { attr, FASTElement, nullableNumberConverter } from '@microsoft/fast-element';
+import { RatingDisplayColor, RatingDisplaySize } from './rating-display.options.js';
+
+/**
+ * The base class used for constructing a fluent-rating-display custom element
+ * @public
+ */
+export class RatingDisplay extends FASTElement {
+  /**
+   * The internal {@link https://developer.mozilla.org/docs/Web/API/ElementInternals | `ElementInternals`} instance for the component.
+   *
+   * @internal
+   */
+  public elementInternals: ElementInternals = this.attachInternals();
+
+  /**
+   * The color of the rating display icons.
+   *
+   * @public
+   * @default `marigold`
+   * @remarks
+   * HTML Attribute: `color`
+   */
+  @attr
+  public color: RatingDisplayColor = RatingDisplayColor.marigold;
```

Comment:
This will reflect the attribute by default which isn't ideal - I'd suggest that instead, we have it be the default _in the stylesheet_ but not set by default in the class.

Reply:
Good point. I've made the change for this and other attributes that had a default value. Thanks.